server/src/repositories/album.repository.ts

```diff
@@ -23,7 +23,7 @@ export class AlbumRepository implements IAlbumRepository {
   getById(id: string, options: AlbumInfoOptions): Promise<AlbumEntity | null> {
     const relations: FindOptionsRelations<AlbumEntity> = {
       owner: true,
-      sharedUsers: true,
+      albumPermissions: { users: true },
```

Comment:
How "deep" these selects go tends to be very crucial for some calls as far as I can tell. I'm not very confident about changing them, but adding `users: true` has worked well in my tests for a bit now.

Reply:
You could replace everywhere you see shared users with permissions/users: true

Reply:
That's what I did, yes.